templates/objective-c/README.md

```diff
@@ -0,0 +1,74 @@
+# PROJECT
+
+[![CI Status](http://img.shields.io/travis/fulldecent/PROJECT.svg?style=flat)](https://travis-ci.org/fulldecent/PROJECT)
```

Comment:
I wonder if we can do something about the `fulldecent` here, and below in the Carthage section